packages/nx/src/tasks-runner/run-command.ts

```diff
@@ -145,7 +145,7 @@ export async function runCommand(
   initiatingProject: string | null,
   extraTargetDependencies: Record<string, (TargetDependencyConfig | string)[]>,
   extraOptions: { excludeTaskDependencies: boolean; loadDotEnvFiles: boolean }
-) {
+): Promise<NodeJS.Process['exitCode']> {
```

Comment:
@FrozenPandaz requested I make this API change here after I demonstrated that I needed `runCommand` to not control the process exit directly